metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -213,6 +213,19 @@ def ensure_top_n_queries_is_not_too_big(cls, v: int) -> int:
             )
         return v
 
+    @pydantic.validator("start_time")
+    def ensure_start_time_coincides_with_bucket_start_time(
```

Comment:
Based on https://github.com/datahub-project/datahub/pull/8562 it seems like this will not be the case by default. Thoughts on making this an INFO log instead?

Reply:
works.